src/lib.js

```diff
@@ -2516,6 +2516,22 @@ export const getStats = async (
     lily: calculateLilyWeight(output),
   };
 
+  /*
+
+    century cake effects
+
+  */
+  const cakes = [];
+  //console.log(userProfile.temp_stat_buffs);
+  for (const cake of userProfile.temp_stat_buffs) {
+    const stat = cake.key.substring(cake.key.indexOf("_") + 1, cake.key.length);
```

Comment:
why not just `const stat = cake.key.replace('cake_', '')` ?

also needs to add an `if (!cake.key.startsWith('cake_')) { continue }` in case in the future century cakes are not the only things inside this temp_stat_buffs